yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -1061,7 +1078,8 @@ def run(self, info):
         self.to_screen('Splitting video by chapters; %d chapters found' % len(chapters))
         for idx, chapter in enumerate(chapters):
             destination, opts = self._ffmpeg_args_for_chapter(idx + 1, chapter, info)
-            self.real_run_ffmpeg([(in_file, opts)], [(destination, self.stream_copy_opts())])
+            out_file_opts = self._set_out_opts(info['ext'], chapter.get('title', ''))
```

Comment:
Dont remove stream_copy_opts

Reply:
_set_out_opts always sets stream_copy_opts()
and then depending on extension , adds more options

Reply:
mb